src/Xamarin.Android.Build.Tasks/Resources/LayoutBinding.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+
+using Android.App;
+using Android.Views;
+
+namespace Xamarin.Android.Design
+{
+	abstract class LayoutBinding
+	{
+		public delegate View OnLayoutViewNotFoundHandler (int resourceId, Type expectedViewType);
```

Comment:
As per earlier discussions, this should *probably* be:

```csharp
public delegate Java.Lang.Object OnLayoutItemNotFound (int resourceId, Type expectedViewType);
```

To help avoid the pains of `android.app.Fragment` vs. support `Fragment` vs. `androidx` Fragment...

Reply:
yeah, I haven't gotten to fragments yet. This is planned